plugins/network/src/triggers/install/install.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+
+	common "github.com/dokku/dokku/plugins/common"
+	proxy "github.com/dokku/dokku/plugins/proxy"
+)
+
+// runs the install step for the network plugin
+func main() {
+	err := common.PropertySetup("network")
+	if err != nil {
+		common.LogFail(fmt.Sprintf("Unable to install the network plugin: %s", err.Error()))
+	}
+
+	apps, err := common.DokkuApps()
+	if err != nil {
+		return
```

Comment:
what do we want to do here though? seems like we should panic or return some error msg, yea?

Reply:
I think it only errors if there are no apps? If thats the case, just ignore...